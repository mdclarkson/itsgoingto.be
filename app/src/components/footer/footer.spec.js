/* global expect */
import React from 'react'
import Footer from 'layouts/footer'
import { shallow } from 'enzyme'

describe('(Layout) Footer', () => {
  describe('(Render)', () => {
    it('matchs snapshot', () => {
      const wrapper = shallow(<Footer />)

      expect(wrapper).toMatchSnapshot()
    })
  })
})
