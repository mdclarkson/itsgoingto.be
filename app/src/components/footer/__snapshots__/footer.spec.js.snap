// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Layout) Footer (Render) matchs snapshot 1`] = `
<div
  className="footer-container"
>
  <div
    className="footer"
  >
    <span
      className="contact"
    >
      <a
        href="mailto:itsgoingtobe@jwgmedia.co.uk"
        target="_top"
      >
        itsgoingtobe@jwgmedia.co.uk
      </a>
    </span>
    <span
      className="jwg_logo"
    >
      <a
        href="http://jwgmedia.co.uk"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          src="jwgmedia.png"
        />
      </a>
    </span>
  </div>
</div>
`;
